components/src/preact/textInput/TextChangedEvent.ts

```diff
@@ -0,0 +1,11 @@
+type LapisTextFilter = Record<string, string | null | undefined>;
+
+export class TextChangedEvent extends CustomEvent<LapisTextFilter> {
```

Comment:
```suggestion
export class TextInputChangedEvent extends CustomEvent<LapisTextFilter> {
```
since the event is called `gs-text-input-changed`.